shell/content/docs/en-us/whats-new.md

```diff
@@ -2,6 +2,33 @@
 title: What's New in 2.6
 ---
 
+
+## New in 2.6.7
+
+### Amazon Billing in the Marketplace
+
+- Now you can purchase support for Rancher through the AWS Marketplace 
```

Comment:
We should still add something here.

Reply:
Per an earlier note, we'll go for:
> You can now purchase support for Rancher directly through the AWS Marketplace:  
https://rancher.com/docs/rancher/v2.6/en/installation/cloud-marketplace/aws/